### DISCUSSION TITLE
Any suggestions on VCS: infinite loading

### DISCUSSION THREAD

**Discussion by cyrilovh** [community/community#44762] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Copilot starts successfully in the Visual Studio code but there are no suggestions. The loading icon runs indefinitely.

Re-installation does not change anything.
I've too change my network (fiber to 4G+)

**Comment by cyrilovh** [comment]
![image](https://user-images.githubusercontent.com/76228887/213430704-8d2931d1-cf2d-42ba-8263-ff90e252c716.png)

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @cyrilovh ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by dawoodjee** [comment]
I'm also having a similar issue

Copilot is working fine on my intel Mac's VS Code, but Copilot Nightly keeps loading after installation. Doesnt finish, doesnt output anything.

<img width="209" alt="image" src="https://user-images.githubusercontent.com/7912989/214579329-3bd32055-bbb5-4c4a-8e5a-db438bf06a67.png">

* in this image i just enabled Copilot (std) as well, Nightly is the next one with the spinner, it stays like that whether or not Copilot std is enabled.

**Reply by dawoodjee** [reply]
Turns out I needed to allow it to connect to Github from one of the bottom left icons (User Icon, or Settings)

<img width="209" alt="image" src="https://user-images.githubusercontent.com/7912989/214580435-88fb53e2-446a-4ef4-971f-a21918d8c6e3.png">
